### PR TITLE
List Component

### PR DESCRIPTION
**Pull request by mehulsatardekar**
Added List Component

Live Preview : https://framer-ui.netlify.app/docs/list.html

### CODE REVIEW

File: docs/list.html [hunk 1]
```diff
@@ -0,0 +1,428 @@
...
+            </div>
+             <div class="flex mt-1 ml-1 flex-align-item-center">
+                <svg width="40" height="32" viewBox="0 0 40 32" aria-hidden="true">
+                    <mask id="32e25648-9cb3-4aba-a168-64581e7aa254" width="52" height="52">
+                        <circle cx="16" cy="16" r="16" fill="white"></circle>
+                        <rect color="black" x="19" y="19" width="16" height="16" rx="8" ry="8"></rect>
+                    </mask>
+                    <foreignObject x="0" y="0" width="32" height="32" mask="url(#32e25648-9cb3-4aba-a168-64581e7aa254)">
+                        <div class="avatarStack-2Dr8S9"><img src="https://cdn.discordapp.com/avatars/709733291946672128/63f8ccaab30e072ae3a184ab142ddc54.webp" alt=" " class="avatar-user-icon" aria-hidden="true"></div>
+                    </foreignObject><svg x="14.5" y="17" width="25" height="15" viewBox="0 0 25 15">
```
Comment (bhushannaikaroskar) [inline docs/list.html]: Hey here alt tag is empty please provide some relevant name to the image or component.

**Comment by prath-dev** [comment]
Clean Code And Elegant UI .

**Comment by prath-dev** [comment]
Looks Good to merge

File: components/list/list.css [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+.notification-container{
+    padding: 1rem 0.5rem 1rem 0.5rem;
+    background-color: var(--text-col-white);
+}
+.notification-img{
+    width: 9.375rem;
+    height: 100%;
+    border-radius: 0.313rem;
+}
```
Comment (bhushannaikaroskar) [inline components/list/list.css]: You can give values in px here

File: components/list/list.css [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+.list-lower-roman{
+    list-style-type: lower-roman;
+}
+
+/* Image section of notification*/
+
+.notification-container{
+    padding: 1rem 0.5rem 1rem 0.5rem;
+    background-color: var(--text-col-white);
```
Comment (bhushannaikaroskar) [inline components/list/list.css]: Same here you have given rem values for lower values. I will suggest to use px

File: components/list/list.css [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+.list-circle{
+    list-style-type: circle;
+    margin-bottom: 0.75rem;
+
+}
+
+.list-disc{
+    list-style-type: disc;
+    margin-bottom: 0.75rem;
+
```
Comment (bhushannaikaroskar) [inline components/list/list.css:36]: Use px here

File: components/list/list.css [hunk 5]
```diff
@@ -0,0 +1,70 @@
+/* @import url('../../components/root.css'); */
+
+@import url('../../components/framer.css');
+
+
+.list-bg{
+    background-color: var(--text-col-white);
+    border-radius: 0.313rem;
```
Comment (bhushannaikaroskar) [inline components/list/list.css]: Use px here as well

File: components/list/list.css (hunk 2 above)
Comment (bhushannaikaroskar) [reply components/list/list.css]: use 3px instead of 0.313rem. anyways px values are too small so better keep it as integers rather than floating number.

**Review by prath-dev — APPROVED** [review]
Its good that you have followed best practices as naming self explanatory css classes and use of semantic tags rest looks good to merge.. UI aslo Looks Clean

**Review by bhushannaikaroskar — APPROVED** [review]
Hey mehul good UI loved it. Suggested some changes please do needful changes before committing. Also I would suggest check padding of card you have used its too low on right and left as compared to top and bottom padding. Rest I am approving given that you have made changes.

File: components/list/list.css (hunk 4 above)
Comment (mehulsatardekar) [reply components/list/list.css:36]: well 0.75 if you convert it into px it is actually 12px

File: components/list/list.css (hunk 5 above)
Comment (mehulsatardekar) [reply components/list/list.css]: will do the need full

**Pull request merged**